### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by ngyihkai**
Ready

### CODE REVIEW

**Comment by ngyihkai** [comment]
Ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (ngocvt-0484) [inline Gemfile:9]: use gem mysql2 please
like sample_app

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
```
Comment (ngocvt-0484) [inline config/database.yml:8]: add to gitignore
like sample app

**Comment by ngyihkai** [comment]
Ready

File: config/credentials.yml.enc [hunk 3]
```diff
@@ -0,0 +1 @@
+sdDqabzS7h4rHHfwV5e88Rf+SEzHW3T+TXEbccnQaI5eKrkW+XiQDctkgaw8zTT4+SfjlrbhL5XDvhldbLdosTWMKeADU7xczxvnO34P2wYx8ApZCDCmylpwCpmvsRVtBttChIV/P5u9RrnLwj1Yz3Tvtz+MNNcMT1VA2qfcQeQv+bme061R62L+EQw1jkxolaDe32YCVqfTYVAXcgJyG5Nseelyuy3X2Z6+3t0G0MGQt+4yNTsGPi2VQwtiOh6UQGOPmln9nM7aIzeNUQ5l1Pgf8qMqAcRg6kEtf9fgHg5/KptjnaSSp7qqDndPIKpw1pDckCoczdTY5wIfzgjeCW4549esXUOhaBEFeiYQshAetOdizvhuzSwoG0rXV3j58RnNcBqnmYfg5pIC9hz0VEriv0IR6pM7NQoT--yeP6fKWfzjNTXLZJ--0Wxo7EUL510P8xH5jbK7bA==
```
Comment (anlvq-1058) [inline config/credentials.yml.enc:1]: use git ignore to ignore this file

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by ngyihkai** [comment]
Ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**